index.html

```diff
@@ -5,21 +5,30 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
     <title>Portfolio</title>
   </head>
   <body>
       <!--work section-->  
-      <header class="navbar">
-        <ul>
-          <li class="navbar-logo">Tadesse J.</li>
-          <li class="navbar-toggle"><i class="fa fa-bars"></i></li>
-          <li class="navbar-links">Portfolio</li>
-          <li class="navbar-links">About</li>
-          <li class="navbar-links">Work</li>
-          <li class="nav-icon" ><i class="fa fa-envelope-o"></i></li>
-        </ul>
-       </header>
+      <header>
+        <nav class="navbar">
+          <a href="#"><h2 class="nav-logo">Tadesse</h2></a>
+          <ul class="nav-menu">
+            <li class="nav-item"><a href="#portfolio" class="nav-link">Portfolio</a><hr class="text-divide"></li>
+            <li class="nav-item"><a href="#about" class="nav-link">About</a><hr class="text-divide"></li>
+            <li class="nav-item"><a href="#contact-form" class="nav-link">Contact</a><hr class="text-divide"></li>
+            <li class="nav-icon" ><i class="fa fa-envelope-o"></i></li>
+          </ul>
```

Comment:
- [ ] You have done well in this project, I really commend you 👍 . I would like you to ensure that you align the menu items to the right and not at the center in the desktop view so that it matches the Figma design template chosen.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/179809873-2c35b343-d9be-4546-a615-3633b4dad143.png)  | ![image](https://user-images.githubusercontent.com/63473584/179809959-b4bb9371-3f30-4de3-8c4a-fc555efe29c1.png) |